### PR TITLE
WIP: Create list of all paratransit agencies

### PR DESCRIPTION
**Pull request by hunterowens**
The following PR is a WIP that parses the various data sources in the README into a list of paratransit agencies with a flag of who qualifies for the mills-alquish-deddeh aka TDA funding. 

TO FIX before merge: 
- [x] move all raw data to gdrive or download scripts 
- [x] add asserts and blackcat data checks

Commits: 
- parse blackcat into a sqlite db
- analyze  num rows
- CTSA list
- download the gsheet
- controller parse into offical mills
- working final list
- add requirements
- rename blackcat
- working makefile
- final dataset
- cleanup
- docs
- notebook parse

### CODE REVIEW

File: catalog.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    args:
+      urlpath: "./intermediate/mills_official.csv"
+  gsheet_trillium_list: 
+    driver: csv
+    description: list from the trillium master and filtered. download using the bash script. 
+    args:
+      urlpath: "./intermediate/offical_list_from_gsheet.csv"
+  ctsa_list: 
+    driver: csv
+    description: the CTSA list of paratransit agencies, processed from the wedny king email.
```
Comment (vykster) [inline catalog.yml]: would you annotate with dates files published or received as appropriate? ie date of email file was _sent by King._

File: controller-parse.ipynb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "df = pd.read_excel('./data/TO_EachDataSet_FY2017-2019_20200716_V1.xlsx',\n",
+    "              sheet_name='TO_OPERATING_DATA')\n",
+    "df.columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to establish a list of paratransit, we filter based on demand responsive hours served to start, leading to 328 rows which will be filtered in the du"
```
Comment (vykster) [inline controller-parse.ipynb]: what is "du"?

File: final/final_combined_paratransit_list.csv [hunk 3]
```diff
@@ -0,0 +1,216 @@
+trillium_agency_name,mills_name,is_mills_act,ctsa_agency_name
```
Comment (vykster) [inline final/final_combined_paratransit_list.csv:1]: Do these agencies have NTD ids?

File: controller-parse.ipynb (hunk 2 above)
Comment (hunterowens) [reply controller-parse.ipynb]: fixed!

File: catalog.yml (hunk 1 above)
Comment (hunterowens) [reply catalog.yml]: updated!

File: final/final_combined_paratransit_list.csv (hunk 3 above)
Comment (hunterowens) [reply final/final_combined_paratransit_list.csv:1]: some of them do, the original trillium files has NTD ids referenced at least. 

I think I should probably rename this file like `deduplicated-crosswalk` or similar, b/c I'm not pulling in anything but the name string in from each of the original datasets

**Comment by hunterowens** [comment]
unless anybody objects, gonna merge this later today

File: README.md [hunk 4]
```diff
@@ -1,2 +1,27 @@
 # paratransit-eligibility-reports
-worked related to the paratransit eligibility reports 
+worked related to the paratransit eligibility reports.
```
Comment (thekaveman) [inline README.md]:
```diff
-worked
+work
```

File: dedupe.ipynb [hunk 5]
```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dedupe and finalize the list \n",
+    "\n",
+    "This notebook dedupes and finalizes the lsit"
```
Comment (thekaveman) [inline dedupe.ipynb]:
```diff
-lsit
+list
```

**Review by thekaveman — COMMENTED** [review]
Some minor typos

File: README.md (hunk 4 above)
Comment (hunterowens) [reply README.md]: thanks!

**Pull request merged**